packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -334,6 +372,48 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     this.subscribeDeviceManagerEvents();
 
     this.callClient.onStateChange(onStateChange);
+    /* @conditional-compile-remove(call-transfer) */
+    if (this.callAgent.kind === 'CallAgent') {
+      const onCallsUpdated = (args: { added: Call[]; removed: Call[] }): void => {
+        if (this.call?.id) {
+          const removedCall = args.removed.find((call) => call.id === this.call?.id);
+          if (removedCall) {
```

Comment:
Can we check the end reason here to see if we know that the removed call is because of the transfer? is there a `endCallReason` for this?

Reply:
The callEndReason when a call is removed by the call agent due to a transfer is: `{code: 0, subCode: 5010}` while the callEndReason when the call is ended by leaving is `{code: 0, subCode: 0}`

Reply:
yeah if the only way we can tell is through the subcodes lets skip it. just wanting to make sure that we don't process out of a call incorrectly.